.github/workflows/ci.yml

```diff
@@ -370,7 +370,7 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
       - name: Run Sentry self-hosted e2e CI
-        uses: getsentry/action-self-hosted-e2e-tests@4ff2dbfdd5c986a4670efaa16d82e460598359eb
+        uses: getsentry/action-self-hosted-e2e-tests@ade3e646ffc3b6a765cfa18ec4ca720f710cb53d
```

Comment:
bump commit here to `711694d0081a834777ca9c77c3f4c322ce7b08c4`

Reply:
Which one is it? Should we use the latest commit on `main` like @hubertdeng123 suggests?

Reply:
Either one will work, but tbh it will be good to confirm that https://github.com/getsentry/action-self-hosted-e2e-tests/commit/711694d0081a834777ca9c77c3f4c322ce7b08c4 does not regress in repos other than self-hosted. Bumped in 867666c972b96f3768ecffeb1938f7f12b0c6e5d.